infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -79,22 +79,32 @@ export class AtatWebApiStack extends cdk.Stack {
     const portfolioDraftId = portfolioDrafts.addResource("{portfolioDraftId}");
     const portfolio = portfolioDraftId.addResource("portfolio");
 
-    const getPortfolioDraftsFn = new lambdaNodejs.NodejsFunction(this, "PortfolioDraftsGetFunction", {
+    // hello world
+    const helloWorldFn = new lambdaNodejs.NodejsFunction(this, "HelloWorldFunction", {
       entry: "applications/portfolioDrafts/index.ts",
       ...sharedFunctionProps,
     });
-    portfolioDrafts.addMethod("GET", new apigw.LambdaIntegration(getPortfolioDraftsFn));
-    // Prevent the GET function from being able to write to DynamoDB (it doesn't need to)
-    table.grantReadData(getPortfolioDraftsFn);
+    portfolioDrafts.addMethod("GET", new apigw.LambdaIntegration(helloWorldFn));
+    table.grantReadData(helloWorldFn);
+
+    // OperationIds from API spec are used to identify functions below
 
     // createPortfolioDraft
-    const createPortfolioDraftFn = new lambdaNodejs.NodejsFunction(this, "createPortfolioDraft", {
+    const createPortfolioDraftFn = new lambdaNodejs.NodejsFunction(this, "CreatePortfolioDraftFunction", {
       entry: "applications/portfolioDrafts/createPortfolioDraft.ts",
       ...sharedFunctionProps,
     });
     portfolioDrafts.addMethod("POST", new apigw.LambdaIntegration(createPortfolioDraftFn));
     table.grantReadWriteData(createPortfolioDraftFn);
 
+    // deletePortfolioDraft
```

Comment:
Can we add a TODO to reduce the boilerplate here? Perhaps we can pass a little config object into a method like this?

```
{
  myNodejsFunction,
  parentRoute,
  "POST",
  table.grantReadWriteData
}
```

Feel free to tell me that I'm a dumb Java guy but I figured we can do something that helps us avoid the dreaded copypasta that can come with a long list of named variables and calling different methods that will all compile but not behave properly.

@kylelaker-ccpo this also brings up the question of how much value we might get out of applying a UT to our stacks here.

Reply:
Absolutely! I am actually planning to work on that today/tomorrow as well. I want to have a `registerDynamoDbInteractionFunction` or something like that with a less awful name that you just pass the API GW resource, the file name, the table, and the HTTP method to and it does some magic for you. The big comment at the top of this section is basically a `// TODO: Reduce copypasta`.

As far as how to test the infra code, working on that too. We should do it. We just need to think about what parts we _actually_ want to test here.

There's also the fact that API GW can "import" an OpenAPI spec, but I don't think that's elegant in the CDK (or CloudFormation generally) today since it's not easy to reference the Lambda integrations, etc. But I am sorta playing around with that idea a bit (but not in any form that will make it to a PR soon).

Reply:
I understood that we wish to:

- reduce boilerplate in `infrastructure/lib/atat-web-api-stack.ts`
- add unit testing against software stack infrastructure